front/app/components/admin/TabbedResource/Tab.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
```

Comment:
It's best to review this component with TabbedResource master version next to it, so you can see what I extracted.